java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -861,6 +861,19 @@ public void expireAt_pexpireAt_and_ttl_with_non_existing_key(BaseClient client)
         assertEquals(-2L, client.ttl(key).get());
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void persist_on_existing_key(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+
+        assertEquals(OK, client.set(key, "foo").get());
+        assertFalse(client.persist(key).get());
+
+        assertTrue(client.expire(key, 10).get());
+        assertTrue(client.persist(key).get());
```

Comment:
check and compare experiation of the key before and after `persist`

Reply:
☝️

Reply:
```
assertEquals(-1L, client.ttl(key).get());
```
This is the check after